waverecoverygui.py

```diff
@@ -0,0 +1,340 @@
+        self.browse_source_button.setFixedWidth(80)
+        self.browse_source_button.clicked.connect(self.browse_source)
+        source_path_layout.addWidget(self.source_path_text)
+        source_path_layout.addWidget(self.browse_source_button)
+
+        dest_path_layout = QHBoxLayout()
+        self.dest_path_label = QLabel('Destination Path:')
+        self.dest_path_text = QTextEdit()
+        self.dest_path_text.setPlaceholderText("Enter destination path")
+        self.dest_path_text.setMaximumHeight(30)
+        self.dest_path_text.setAcceptRichText(False)
+        self.browse_dest_button = QPushButton('Browse')
+        self.browse_dest_button.setFixedWidth(80)
+        self.browse_dest_button.clicked.connect(self.browse_dest)
+        dest_path_layout.addWidget(self.dest_path_text)
+        dest_path_layout.addWidget(self.browse_dest_button)
+
+        self.sample_rate_label = QLabel('Sample Rate:')
+        self.sample_rate_combobox = QComboBox()
+        self.sample_rate_combobox.setEditable(True)
+
+        # Add predefined sample rates to the combobox
+        sample_rates = ['8000', '44100', '48000', '88200', '96000', '192000']
+        self.sample_rate_combobox.addItems(sample_rates)
+
+        # Set the default/pre-selected sample rate
+        default_sample_rate = '44100'
+        self.sample_rate_combobox.setCurrentText(default_sample_rate)
+
+        # Set validator to allow only numeric input
+        validator = QIntValidator()
+        self.sample_rate_combobox.setValidator(validator)
+
+        self.sample_rate_label = QLabel('Sample Rate:')
+        self.sample_rate_spinbox = QSpinBox()
+        self.sample_rate_spinbox.setMinimum(1)
+        self.sample_rate_spinbox.setMaximum(999999)
+        self.sample_rate_spinbox.setValue(44100)
+
+        self.bits_per_sample_label = QLabel('Bits Per Sample:')
+        self.bits_per_sample_combobox = QComboBox()
+        self.bits_per_sample_combobox.setEditable(True)
+
+        # Add predefined bits per sample to the combobox
+        bits_per_sample_values = ['8', '16', '24', '32']
+        self.bits_per_sample_combobox.addItems(bits_per_sample_values)
+
+        # Set the default/pre-selected bits per sample
+        default_bits_per_sample = '16'
+        self.bits_per_sample_combobox.setCurrentText(default_bits_per_sample)
+
+        # Set validator to allow only numeric input
+        validator = QIntValidator()
+        self.bits_per_sample_combobox.setValidator(validator)
+
+        self.channels_label = QLabel('Channels:')
+        self.channels_combobox = QComboBox()
+        self.channels_combobox.setEditable(True)
+
+        # Add predefined channel values to the combobox
+        channel_values = ['1', '2']
+        channel_labels = ['1 (Mono)', '2 (Stereo)']
+        self.channels_combobox.addItems(channel_labels)
+
+        # Set the default/pre-selected number of channels
+        default_channels = '1'

```

Comment:
Shouldn't this be `1 (Mono)`? Otherwise it is inconsistent with the state after you select `1 (Mono)` from the dropdown.